recipes/libdispatch/all/conanfile.py

```diff
@@ -58,6 +55,8 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "share"))
 
     def package_info(self):
-        self.cpp_info.libs = tools.collect_libs(self)
         if self.settings.os == "Linux":
+            self.cpp_info.libs = ["dispatch", "BlocksRuntime"]
```

Comment:
I thought blocks runtime was a separate project. 😖 Apple is weird.

https://github.com/apple/swift-corelibs-libdispatch/blob/f13ea5dcc055e5d2d7c02e90d8c9907ca9dc72e1/cmake/modules/FindBlocksRuntime.cmake

https://compiler-rt.llvm.org/

https://github.com/mackyle/blocksruntime